tidb-cloud/third-party-monitoring-integrations.md

```diff
@@ -38,3 +38,7 @@ For the detailed integration steps and a list of metrics that Datadog tracks, re
 With the Prometheus and Grafana integration, you can get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards.
 
 For the detailed integration steps and a list of metrics that Prometheus tracks, see [Integrate TiDB Cloud with Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
+
+### New Relic integration
+
+...
```

Comment:
@lilyjazz Please add relevant information for new relic.

Reply:
done